### PR TITLE
Bug 1205212 - Implement new Pin Card r=gmarty

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=da3612b55ad4ee3e43d3c69f488037515d92485c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f771f466c70c1954bedeab09819e4f25464125e7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=929e2d22bedea1c490475c3b83861f6e986fbcc2

File: shared/elements/gaia_pin_card/script.js [hunk 1]
```diff
@@ -88,7 +88,9 @@ window.GaiaPinCard = (function(win) {
   var template = document.createElement('template');
   template.innerHTML =
     `<article class="pin-card no-content">
-      <i></i>
+      <div class="icon-container">
+        <icon></icon>
```
Comment (gmarty) [inline shared/elements/gaia_pin_card/script.js]: What's the `<icon>` HTML element? Seems neither standard, nor a web component.

Comment (albertopq) [inline shared/elements/gaia_pin_card/script.js]: I remember reading that using <i> for icons was not the best idea (semantically),  but not sure where the icon tag came from. Let's leave it as it was. Thanks!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7fc0090d4fe49be548ba45e672b6ad3d2ab4e7c9

**Pull request merged**